.github/workflows/snapshot.yaml

```diff
@@ -8,7 +8,6 @@ jobs:
   release:
     permissions:
       id-token: write # aws-actions/configure-aws-credentials@v4.0.1
-    if: github.repository == 'aws/karpenter-provider-aws'
```

Comment:
What's the reasoning behind removing the repo check? Is it implied that we're in the right repo?